### PR TITLE
[13.0][FIX] account_payment_term_partner_holiday: Allow action_post() from moves with multi-record

### PR DESCRIPTION
**Pull request by victoralmau**
Allow `action_post()` from moves with multi-record

Related to https://github.com/OCA/account-payment/pull/523#discussion_r925182193

Please @chienandalu and @LudLaf can you review it?

@Tecnativa TT37271

### CODE REVIEW

File: account_payment_term_partner_holiday/models/account_move.py [hunk 1]
```diff
@@ -41,4 +41,5 @@ def action_post(self):
         """
         for item in self:
             _item = item.with_context(move_partner_id=item.partner_id.id)
-            return super(AccountMove, _item).action_post()
+            res = super(AccountMove, _item).action_post()
+        return res
```
Comment (chienandalu) [inline account_payment_term_partner_holiday/models/account_move.py]: This is confusing as we're overwriting the result value in every loop. I know the original method always returns `True`. Maybe that's what we should return no matter what.

**Review by chienandalu — APPROVED** [review]
:smile: :+1:

**Comment by LudLaf** [comment]
Looks good to me

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**